reco_utils/recommender/deeprec/deeprec_utils.py

```diff
@@ -389,6 +388,14 @@ def cal_metric(labels, preds, metrics):
             pred[pred < 0.5] = 0
             f1 = f1_score(np.asarray(labels), pred)
             res["f1"] = round(f1, 4)
+        elif metric == 'mae':
+            res['mae'] = mean_absolute_error(np.asarray(labels), np.asarray(preds))
+        elif metric == 'rsquare':
+            res['rsquare'] = r2_score(np.asarray(labels), np.asarray(preds))
```

Comment:
I think np.asarray is not needed, if I'm not mistaken it is slower and the result is the same without it